include/libtorrent/alert_types.hpp

```diff
@@ -782,18 +782,23 @@ TORRENT_VERSION_NAMESPACE_2
 #endif
 	};
 
-	// This alert is posted every time an outgoing peer connect attempts succeeds.
+	// This alert is posted every time an incoming or outgoing peer connect attempts succeeds.
```

Comment:
this description could be more precise. Incoming connections aren't "attempts", and I think it's important to point out that this is not posted when we accept an incoming connection, but when an incoming connection handshakes and is associated with a torrent